tests/virt_autotest/reboot_and_wait_up_upgrade.pm

```diff
@@ -17,9 +17,38 @@ use testapi;
 use base "reboot_and_wait_up";
 
 sub run {
-    my $self    = shift;
-    my $timeout = 600;
+    my $self = shift;
+
+    #defaultly offline upgrade
```

Comment:
defaultly?

Reply:
may be initially is better, rather than no other choice, choose this as default